pkg/acceptance/testing.go

```diff
@@ -24,6 +24,7 @@ const (
 	TestSchemaName     = "terraform_test_schema"
 	TestWarehouseName  = "terraform_test_warehouse"
 	TestWarehouseName2 = "terraform_test_warehouse_2"
+	TestUserName       = "terraform_test_user"
```

Comment:
Please do not create the user in this global setup but create one in PreConfig or by simply adding `resource snowflake_user test_user {...}` to the config used in the test.